dojo/importers/base_importer.py

```diff
@@ -38,7 +38,7 @@
     Test_Import_Finding_Action,
     Test_Type,
     Tool_Configuration,
-    Vulnerability_Id,
+    Vulnerability_Id,  # noqa: F401
```

Comment:
```suggestion
```
I do not understand usage of `noqa` here. If `Vulnerability_Id` is not used in the rest of the file, import might be simply removed.

Reply:
It is used in a bunch of decorators in tests like in TestImporterUtils and the linter doesn't understand the decorator:

@patch('dojo.importers.base_importer.Vulnerability_Id', autospec=True)
    def test_handle_no_vulnerability_ids_references_and_cve(self, mock):
    
And I readily admit that I am unsure why or even how these tests are supposed to work....